StudioTBA/CoronaIO
# Gnoriega27/human flee

## What issue is this PR solving?
Closes: #7 

## Description of change
A human flee to a shelter.

## How to reproduce the changes
1. Start the "Sandbox" scene
1. Make sure the "TestHumanAlertZombie" object is in the scene and enabled.
